app/src/main/res/layout-land/app_language_fragment.xml

```diff
@@ -46,6 +52,7 @@
         android:overScrollMode="never"
         android:paddingBottom="96dp"
         android:scrollbars="none"
+        app:list="@{viewModel.recyclerViewAppLanguageList}"
```

Comment:
This should come in last
`        app:list="@{viewModel.recyclerViewAppLanguageList}" />`

Reply:
Done